aspnetcore/signalr/client-features.md

```diff
@@ -0,0 +1,28 @@
+---
+title: SignalR client features
+author: bradygaster
+description: Learn which features are supported by the various ASP.NET Core SignalR clients.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: bradyg
+ms.custom: mvc
+ms.date: 09/18/2019
+uid: signalr/client-features
+---
+# ASP.NET Core SignalR client features
+
+## Feature distribution
+
+The table below illustrates the matrix of features and their support across the clients that offer real-time support to various platforms. 
```

Comment:
The table below shows the features and support for the clients that offer real-time support.

Reply:
Less is more. My version will MT much better.